packages/server/src/validator/utils.ts

```diff
@@ -14,7 +14,11 @@ export function validateParam(index: number | string, param: any, validation: an
     throw predefined.MISSING_REQUIRED_PARAMETER(index);
   }
 
-  if (param != null && Array.isArray(paramType)) {
+  if (param === null) {
+    throw predefined.INVALID_PARAMETER(index, `The value passed is not valid: ${param}.`);
+  }
+
+  if (param !== undefined && Array.isArray(paramType)) {
```

Comment:
Why are we allowing `undefined` as a parameter, but not `null`?

Reply:
Null is allowed only for to address in transaction objects and filter objects, which is cerified in the if below, so it makes sense that any other param equal to null should not be allowed. Undefined is allowed for unrequired params, so I've changed the logic a little bit to check it above in an else after the check for requiredIsMissing